### PR TITLE
upgrade to pylexibank 2.1

### PR DESCRIPTION
**Pull request by xrotwang**

### CODE REVIEW

File: src/lexibank_hantganbangime/cldf/requirements.txt [hunk 1]
```diff
@@ -1,53 +1,48 @@
...
-requests==2.22.0
-rfc3986==1.3.2
-segments==2.0.2
-six==1.12.0
-SQLAlchemy==1.3.10
-tabulate==0.8.5
+pyclts==2.0.0
+pyconcepticon==2.3.0
+pycountry==18.12.8
+-e git+https://github.com/glottolog/pyglottolog.git@4334cb3539a6c40496e5f832c6ad9c532470a7e4#egg=pyglottolog
```
Comment (chrzyki) [inline src/lexibank_hantganbangime/cldf/requirements.txt:31]: Ideally pyglottolog==2.2.1 or similar?

**Comment by LinguList** [comment]
sorry, I got confused by looking at zenodo and did not see you asked me to review, but I think this is fine in this form.

**Comment by shh-dlce-zenodo** [comment]
You did merge as zenodo user, though.

**Comment by LinguList** [comment]
Yes, that's what I meant by "confused".

**Pull request merged**